testsuite/MDAnalysisTests/lib/test_pkdtree.py

```diff
@@ -31,21 +31,29 @@
 
 
 # fractional coordinates for data points
-f_dataset = np.array([[0.2, 0.2, 0.2],  # center of the box
-                      [0.5, 0.5, 0.5],
-                      [0.11, 0.11, 0.11],
-                      [1.1, -1.1, 1.1],  # wrapped to [1, 9, 1]
-                      [2.1, 2.1, 0.3]],  # wrapped to [1, 1, 3]
-                     dtype=np.float32)
-
-
-@pytest.mark.parametrize('b, cut, result', (
-                         (None, 1.0,
-                          'Donot provide cutoff distance'
-                          ' for non PBC aware calculations'),
-                         ([10, 10, 10, 90, 90, 90], None,
-                          'Provide a cutoff distance with'
-                          ' tree.set_coords(...)')))
+f_dataset = np.array(
+    [
+        [0.2, 0.2, 0.2],  # center of the box
+        [0.5, 0.5, 0.5],
+        [0.11, 0.11, 0.11],
+        [1.1, -1.1, 1.1],  # wrapped to [1, 9, 1]
+        [2.1, 2.1, 0.3],
+    ],  # wrapped to [1, 1, 3]
```

Comment:
This comment is in the wrong place now I think

Reply:
Yes, well spotted. I'll double check those.